lib/testing/src/lib/process-services/pages/process-list.page.ts

```diff
@@ -45,4 +45,8 @@ export class ProcessListPage {
     checkContentIsNotDisplayedByColumn(column: string, processName: string): Promise<void> {
         return this.dataTable.checkContentIsNotDisplayed(column, processName);
     }
+
+    async selectRow(processName): Promise<void> {
+        await this.dataTable.selectRow('Name', processName);
```

Comment:
what is name? the name of the column ? what if the column is not there?

Reply:
It makes more sense for me also to have there a variable instead of a string but I was writing '_Name_' there because:
- I thought that _"Name"_ is a default Process Services column name and it would not (or at least it shouldn't) change;
- I saw the same method in _TaskListPage_:
```
    selectRow(taskName): Promise<void> {
        return this.dataTable.selectRow('Name', taskName);
    }
```

What should I do in this case?

Reply:
Method name changed to `selectRowByName`